src/ocamlorg_frontend/package_intf.ml

```diff
@@ -1,16 +1,29 @@
+type version = Latest | Version of string
+
 type package = {
   name : string;
   description : string;
   license : string;
-  version : string;
+  version : string option;
+      (* None = we are looking at the latest version of the package on a
+         /latest-URL, Some v = we are looking at version v *)
   versions : string list;
+  latest_version : string;
   tags : string list;
   rev_deps : string list;
   authors : Ood.Opam_user.t list;
   maintainers : Ood.Opam_user.t list;
   publication : float;
 }
 
+let exact_version package =
+  match package.version with None -> package.latest_version | Some v -> v
+
+let render_package_version package =
+  match package.version with
+  | None -> Fmt.str "%s (latest)" package.latest_version
+  | Some v -> v
```

Comment:
I don't think function definitions should go into an `_intf.ml` file. To my understanding, they're meant to host type definitions only.

Reply:
Ok that makes sense. This module is essentially "what a package looks like to the templates", so renaming to `package.ml` would make sense.